docs/operators/kafkaInputOperator.md

```diff
@@ -11,6 +11,8 @@ Kafka is a pull-based and distributed publish subscribe messaging system, topics
 nodes. Kafka input operator is needed when you want to read data from multiple
 partitions of a Kafka topic in parallel in an Apex application.
 
+### 0.8 Version of Kafka Input Operator
+
 ### AbstractKafkaInputOperator
```

Comment:
Since both have the same name, add the the package name in brackets for both so users can distinguish them.